test/FastTests/Server/Documents/Versioning/Versioning.cs

```diff
@@ -458,11 +458,16 @@ public class Versioning : RavenTestBase
                 Assert.Equal(21, statistics.CountOfDocuments);
                 Assert.Equal(20, statistics.CountOfRevisionDocuments);
 
-                using (var context = DocumentsOperationContext.ShortTermSingleUse(database))
-                using (var tx = context.OpenWriteTransaction())
+                if (useConsole)
+                {
+                    new AdminJsConsole(database).ApplyScript(new AdminJsScript
+                    {
+                        Script = "database.DocumentsStorage.VersioningStorage.Operations.DeleteRevisionsBefore('Users', new Date());"
```

Comment:
Construct a date using JS and jint will convert it to DateTime.
I guess `new Date(2017, 6, 3)` would work.